tests/conftest.py

```diff
@@ -122,10 +122,11 @@ def verify_kubeapi_service(host):
     _verify_kubeapi_service(host)
 
 
-@then(parsers.parse(
-    "we have {pods_count:d} running pod labeled '{label}' "
-    "in namespace '{namespace}' on node '{node}'"
-))
+@then(parsers.re(
+    r"we have (?P<pods_count>\d+) running pod labeled '(?P<label>[^']+)' "
+    r"in namespace '(?P<namespace>[^']+)'( on node '(?P<node>[^']+)')?"),
```

Comment:
minor: use a non-capturing group for the external parenthesis: `(?: on node '(?P<node>[^']+)')?`

Reply:
done